library/aes.c

```diff
@@ -397,8 +397,14 @@ static int aes_init_done = 0;
 static void aes_gen_tables( void )
 {
     int i, x, y, z;
-    int pow[256];
-    int log[256];
+#if !defined(MBEDTLS_AES_FEWER_TABLES)
+    uint32_t *pow = RT2;
```

Comment:
I understand the rationale about reusing existing tables, but this might confuse users, since RT ( Reverse tables) is not for power and log calculations.

Reply:
Yeah you are right that re-usage can be confusing. But maybe I should add there comment and explain this. That this is  only for "optimization" and here we treat it only as a tmp helper